students/shayna_andersonhill/grid_printer_exercise/grid_printer.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python3
+#Shayna Anderson-Hill
+#Script to write a function that draws a grid
+#01-22-2018
+
+plus = '+'
+minus = '-'
+space = ' '
+line = '|'
+
+#Naive Answer
+print('Part One')
+print(plus, (minus+space)*4,plus, (minus+space)*4,plus)
+print(line, space*8, line, space*8, line)
+print(line, space*8, line, space*8, line)
+print(line, space*8, line, space*8, line)
+print(line, space*8, line, space*8, line)
+print(plus, (minus+space)*4,plus, (minus+space)*4,plus)
+print(line, space*8, line, space*8, line)
+print(line, space*8, line, space*8, line)
+print(line, space*8, line, space*8, line)
+print(line, space*8, line, space*8, line)
+print(plus, (minus+space)*4,plus, (minus+space)*4,plus)
```

Comment:
Can you think of how you might write this with a `for` loop? HINT: nested for loops :)

Reply:
Can you review my nested for loop solution. Would it be better if I found a way to include the last print statement as part of the loop?